crates/bevy_gizmos/src/lib.rs

```diff
@@ -116,7 +127,7 @@ impl Plugin for GizmoPlugin {
 
         render_app.add_systems(
             Render,
-            prepare_line_gizmo_bind_group.in_set(RenderSet::PrepareBindGroups),
+            (prepare_line_gizmo_bind_group).in_set(RenderSet::PrepareBindGroups),
```

Comment:
Is there some reason for this change? We don't need the tuple

Reply:
There isn't. That is an artefact from a previous attempt. I'll remove it now.